src/vmm/src/vmm_config/logger.rs

```diff
@@ -61,6 +61,22 @@ pub struct LoggerConfig {
     pub show_log_origin: bool,
 }
 
+fn default_level() -> LoggerLevel {
+    LoggerLevel::Warning
+}
+
+impl LoggerConfig {
+    /// Obtain a default LoggerConfig representation.
+    pub fn new(log: PathBuf) -> LoggerConfig {
```

Comment:
We should first talk with Firecracker customers to find out if these are good defaults. This way we can minimize the updates required on the logger.